source/Target/LanguageRuntime.cpp

```diff
@@ -344,24 +344,8 @@ LanguageRuntime::GuessLanguageForSymbolByName(Target &target,
     return eLanguageTypeUnknown;
 }
 
-bool
-LanguageRuntime::IsSymbolAnyRuntimeThunk(ProcessSP process_sp, Symbol &symbol)
-{
-  if (!process_sp)
-    return false;
-    
-  enum LanguageType languages_to_try[] = {
-      eLanguageTypeSwift, eLanguageTypeObjC, eLanguageTypeC_plus_plus};
-
-  LanguageRuntime *language_runtime;
-  for (enum LanguageType language : languages_to_try) {
-    language_runtime = process_sp->GetLanguageRuntime(language);
-    if (language_runtime) {
-        if (language_runtime->IsSymbolARuntimeThunk(symbol))
-          return true;
-    }
-  }
-  return false;
+bool LanguageRuntime::IsSymbolAnyRuntimeThunk(Symbol &symbol) {
+  return SwiftLanguageRuntime::IsSymbolARuntimeThunk(symbol);
```

Comment:
Since `LanguageRuntime::IsSymbolAnyRuntimeThunk()` is a swift-lldb internal addition, the LHS really looks like premature generalization to me. Either we add no-op support in llvm.org or we just don't bother, like in this patch. I'm leaning towards the latter.